docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -36,6 +36,98 @@ To start a Besu node run:
 besu [OPTIONS] [COMMAND]
 ```
 
+### `api-gas-price-blocks`
+
+=== "Syntax"
+
+    ```bash
+    --api-gas-price-blocks=<INTEGER>
+    ```
+
+=== "Example"
+
+    ```bash
+    --api-gas-price-blocks=1000
```

Comment:
Are we sure this `1000` blocks example is not going to create any issue? It's a lot. The more blocks the more variation of gas price you can get. I tend to think that if user has to change the value to override the default, it may be with a lower value instead of a higher one to be closer to the latest gas price. A higher one would only provide a value closer to the long term median and I don't think there's much interest for that as the goal is to be able to accurately evaluate the gas price to send a tx. Smoothing the latests changes a bit is important, but having a too wide range will provide more inaccurate values.

Reply:
Good point.  Updated to 50.